tests/test_extract.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from etl.extract import extract_csv_data, extract_fund_info
+
+
+def test_extract_fund_info() -> None:
```

Comment:
Good text of this utility, though would be good to parameterize with more test inputs.